### DISCUSSION TITLE
BTAppContextSwitcher has no member setReturnUrlScheme()

### DISCUSSION THREAD

**Discussion by leej56871** [braintree/braintree_ios#1070] [General]
Hi, Im a student newly learning about Swift.
While I was exploring APIs and guides in Braintree, it told me to put

'BTAppContextSwitcher.setReturnUrlScheme()'

in my AppDelegate, and definitely getting an error since BTAppContextSwitcher doesn't have such method
https://github.com/braintree/braintree_ios/blob/6.2.0/Sources/BraintreeCore/BTAppContextSwitcher.swift#L13

As since that is one of prerequisite for implementing Braintree API for iOS, I am blocked from the beginning.
Please teach me alternative way to resolve such problem.

Thank you so much.

**Comment by scannillo** [comment]
Hello @leej56871 - this is a great find. This looks like a method that we did not port over to iOS V6. For now, you can just set this:

```
BTAppContextSwitcher.sharedInstance.returnURLScheme = "com.some.scheme"
```

But we can update V6 to offer a this setter helper method and make this a little cleaner. We are also in the process of updating our [developer documentation ](https://developer.paypal.com/braintree/docs/)for V6.

**Comment by scannillo** [comment]
👋 We have a PR up addressing this if you'd like to follow along - https://github.com/braintree/braintree_ios/pull/1077
